src/cmap/connection.ts

```diff
@@ -316,6 +316,8 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
       this[kDelayedTimeoutId] = null;
     }
 
+    this[kStream].setTimeout(0);
```

Comment:
We can't only reset the time out here, because monitoring connections emit multiple messages for a single command.

Maybe we should move this logic into the if statement that checks for `moreToCome`.  That statement is already designed to handle the streaming protocol, so we can reset the socket time out there if needed